projects/libsass/data_context_fuzzer.cc

```diff
@@ -0,0 +1,32 @@
+#include "sass.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  char* sass_data = (char*) malloc(sizeof(char) * size + 1);
+  if (sass_data == NULL) return 0;
+
+  memcpy(sass_data, data, size);
+  sass_data[size] = '\0';
```

Comment:
nit: Since we are using CXX to compile this file, I would just use `std::string`

Reply:
Ah yeah, I did a double-take when I looked at this as well: the library takes ownership of the input buffer, so using std::string causes a double free.